### DISCUSSION TITLE
How to get the Differences when Migration indicates drift

### DISCUSSION THREAD

**Discussion by ebaer-payix** [prisma/prisma#7658] [Q&A] [answered]
Hi,

Is there any way to get the details about the differences between the Current Database and the Migration, when the Migration indicates that there has been Drift ?

If so, how ?

Thanks

eb

**Comment by janpio** [comment, marked as answer]
Yes, the next release should have much better information here, see the 2 closed issues about drift here: https://github.com/prisma/prisma/issues?q=is%3Aissue+sort%3Aupdated-desc+drift+milestone%3A2.25.0+is%3Aclosed

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
